seqr/views/apis/summary_data_api_tests.py

```diff
@@ -701,12 +731,253 @@ def test_sample_metadata_export(self, mock_google_authenticated):
+            'proband_id': 'NA20889',
+            'maternal_id': '',
+            'paternal_id': '',
+            'other_individual_ids': 'NA20870; NA20888',
+            'individual_count': 3,
+            'family_structure': 'other',
+            'family_history': 'Yes',
+            'genes': 'DEL:chr1:249045487-249045898; OR4G11P',
+            'pmid_id': None,
+            'phenotype_description': None,
+            'analysisStatus': 'Q',
+            'analysis_groups': '',
+            'consanguinity': 'Unknown',
+        })
+
+        # Test all projects
+        all_projects_url = reverse(family_metadata, args=['all'])
+        response = self.client.get(all_projects_url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(list(response_json.keys()), ['rows'])
+        all_project_families = [
+            'F000001_1', 'F000002_2', 'F000003_3', 'F000004_4', 'F000005_5', 'F000006_6', 'F000007_7', 'F000008_8',
+            'F000009_9', 'F000010_10', 'F000011_11', 'F000012_12', 'F000013_13']
+        self.assertListEqual(sorted([r['familyGuid'] for r in response_json['rows']]), all_project_families)
+        test_row = next(r for r in response_json['rows'] if r['familyGuid'] == 'F000003_3')
+        self.assertDictEqual(test_row, {
+            'projectGuid': 'R0001_1kg',
+            'internal_project_id': '1kg project nåme with uniçøde',
+            'familyGuid': 'F000003_3',
+            'family_id': '3',
+            'displayName': '3',
+            'solve_status': 'Unsolved',
+            'actual_inheritance': '',
+            'date_data_generation': '2017-02-05',
+            'data_type': 'WES',
+            'other_individual_ids': 'NA20870',
+            'individual_count': 1,
+            'family_structure': 'singleton',
+            'genes': '',
+            'pmid_id': None,
+            'phenotype_description': None,
+            'analysisStatus': 'Q',
+            'analysis_groups': 'Accepted; Test Group 1',
+            'consanguinity': 'Unknown',
+            'condition_id': 'OMIM:615123',
+            'known_condition_name': '',
+            'condition_inheritance': 'Unknown',
+        })
+
+        # Test analyst access
+        self.login_analyst_user()
+        response = self.client.get(all_projects_url)
+        self.assertEqual(response.status_code, 200)
+        self.assertListEqual(
+            sorted([r['familyGuid'] for r in response.json()['rows']]), all_project_families + self.ADDITIONAL_FAMILIES)
+
+        # Test empty project
+        empty_project_url = reverse(family_metadata, args=['R0002_empty'])
+        response = self.client.get(empty_project_url)
+        self.assertEqual(response.status_code, 200)
+        self.assertDictEqual(response.json(), {'rows': []})
+
+    def test_variant_metadata(self):
+        url = reverse(variant_metadata, args=[PROJECT_GUID])
+        self.check_collaborator_login(url)
```

Comment:
copy pasted from report_api_tests, only change is changing this from `check_analyst_login` and then adding analyst login back in later in the test